tests/input_files/mixture_solid_material_viscoelasthyper.4C.yaml

```diff
@@ -1,70 +1,163 @@
-# Test of the solid viscoelastic material within the mixture framework
 IO:
-  STRUCT_STRAIN: gl
-  STRUCT_STRESS: 2pk
+  STRUCT_STRESS: Yes
+  STRUCT_STRAIN: Yes
 IO/RUNTIME VTK OUTPUT:
-  OUTPUT_DATA_FORMAT: ascii
   INTERVAL_STEPS: 1
-  EVERY_ITERATION: no
+  OUTPUT_DATA_FORMAT: ascii
 IO/RUNTIME VTK OUTPUT/STRUCTURE:
```

Comment:
Regarding the conversion mechanism (here demonstrated for old yaml files):

I made sure that sections are ordered the same way after conversion. Within a section, I did not bother to keep this order. The order instead corresponds the order in the `InputSpec`. Also note how we automatically removed default values (here: `EVERY_ITERATION: no`).